apps/src/gamelab/AnimationTab/PiskelEditor.jsx

```diff
@@ -0,0 +1,70 @@
+/** @file Component wrapping embedded Piskel editor */
+// PISKEL_DEVELOPMENT_MODE is a build flag.  See Gruntfile.js for how to enable it.
+/* global PISKEL_DEVELOPMENT_MODE */
+import React from 'react';
+import {connect} from 'react-redux';
+import { METADATA_SHAPE } from '../animationMetadata';
+import {MessageType} from '@code-dot-org/piskel';
```

Comment:
Space on line 6 but not on lines 5 or 7? (My own preference is to have spaces, tho I think yours is usually not to)